src/TimeZones.jl

```diff
@@ -30,14 +32,36 @@ export TimeZone, @tz_str, istimezone, FixedTimeZone, VariableTimeZone, ZonedDate
     # ranges.jl
     guess
 
-const PKG_DIR = dirname(@__DIR__)
-const DEPS_DIR = joinpath(PKG_DIR, "deps")
+# Compile the deps files contents into the precompiled module for relocatability.
+const DEPS_CONTENTS = Dict{String,String}()
+let deps = joinpath(@__DIR__, "..", "deps")
+    for (root, dirs, files) in walkdir(deps), file in files
+        path = joinpath(relpath(root, deps), file)
+        include_dependency(joinpath(deps, path))
+        DEPS_CONTENTS[path] = read(joinpath(deps, path), String)
+    end
+end
 
 # TimeZone types used to disambiguate the context of a DateTime
 # abstract type UTC <: TimeZone end  # Already defined in the Dates stdlib
 abstract type Local <: TimeZone end
 
+const pkg_version = VersionNumber(TOML.parsefile(joinpath(dirname(@__DIR__), "Project.toml"))["version"])
+
 function __init__()
+    global DEPS_DIR = @get_scratch!("timezones-$VERSION-$pkg_version")
```

Comment:
May be able to use the tzdata version

Reply:
Can do, is the cache folder structure likely to remain the same between package versions? If not then using just `tzdata_version` as the scratch key may cause issues, perhaps.

Reply:
It's probably safest to include Julia version, package version, and tzdata version. It also just occurred to me that users are allowed to [re-compile the time zones to support dates in the far future](https://juliatime.github.io/TimeZones.jl/stable/faq/#future_tzs-1). I'll need to think more about this.